pages/_categoryCode/index.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+<div class="page page--category">
+  <article-category-banner/>
+
+  <!-- Category leftside wrapper -->
+  <div class="leftside-wrapper">
+    <main class="main-content">
+      <article-card-list :class="[ 'article-card-list--horizontal', !isMobile ? 'article-card-list--big-main-post' : '' ]" :articles="list"/>
+    </main>
+    <aside class="aside-down">
+    </aside>
+  </div>
+
+  <!-- Category aside right -->
+  <aside class="aside-right">
+    <categories-widget/>
+  </aside>
+</div>
+</template>
+
+<script>
+import ArticleCategoryBanner from '~/components/article/ArticleCategoryBanner'
+import CategoriesWidget from '~/components/widgets/CategoriesWidget'
+import ArticleCardList from '~/components/article/ArticleCardList'
+import { mapGetters } from 'vuex'
+import _ from 'lodash'
+
+export default {
+  layout: 'common',
+  components: {
+    ArticleCategoryBanner,
+    CategoriesWidget,
+    ArticleCardList
+  },
+  computed: {
+    ...mapGetters({
+      categoryCodeList: 'view/articles/categoryCodeList',
+      isMobile: 'general/isMobile'
+    }),
+    list () {
+      return _.map(this.categoryCodeList, code => this.$store.getters[ 'resources/articles' ][code])
+    }
+  },
+  fetch ({ store, params }) {
+    if (params.categoryCode === 'wszystkie') {
```

Comment:
'wszystkie' should be const

Reply:
It's used only once I cannot get why should I create const for it. Should import the name from settings or what?

Reply:
We using it three times in whole codebase, and and usage is already a declared const, in my opinion it should be strict const with uppercased name in seperate 'consts' file where we could add more like this: `const CATEGORY_CODE_ALL = 'wszystkie'`

Reply:
Where then the const file should be placed?

Reply:
🗡  models/Category/CategoryInterface

Reply:
That was not helpful brother xD

Reply:
I would keep this alongside logic connected with category, but of course we don't have this yet

Reply:
It's better idea i like it. What do you think @zrebcu?

Reply:
I think it's not config. Maybe new directory `constants/common.js`?

Reply:
Then the constants folder will be introduced.

Reply:
soon we'll end up in 20+ top level directories, I don't think it's good way to organize project..
Imagine we'll have some specific functions for articles, we would create 'helpers/aricleHelper'? etc?
Why not create single directory for domain logic? Like `domain/articles` or similar? Then there could be functions, objects, consts related with `articles`